citations-impl/impl/src/java/org/sakaiproject/citation/impl/CitationListAccessServlet.java

```diff
@@ -363,6 +364,7 @@ protected void handleViewRequest(HttpServletRequest req, HttpServletResponse res
     			String href = citation.hasPreferredUrl() ? citation.getCustomUrl(citation.getPreferredUrlId()) : citation.getOpenurl();
     			
     			out.println("\t\t<td headers=\"details\">");
+    			out.println("\t\t\t<div class=\"detailsDiv\"><div class=\"titleDiv\"><a href=\"" + Validator.escapeHtml(href) + "\"><img id=\"" + citation.getCitationProperty("isnIdentifier") + "\" src=\"\" class=\"citationThumbnail\"></a><a href=\"" + Validator.escapeHtml(href) + "\" target=\"_blank\">" + Validator.escapeHtml( (String)citation.getCitationProperty( Schema.TITLE, true ) ) + "</a>");
```

Comment:
You mean if nothing loads initially and then the image pops in and page jumps? Yes I'll see if there's a problem with that and create a Jira to fix it if so